.github/pull_request_template.md

```diff
@@ -7,9 +7,4 @@
 <!--- Why is this change required? What problem does it solve? -->
 
 
-## Scope
-<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-
```

Comment:
**Correctness**: Unrelated to upgrading Smithy. This should be moved to a separate commit.